test/e2e_node/apparmor_test.go

```diff
@@ -146,7 +146,9 @@ func loadTestProfiles() error {
 
 func runAppArmorTest(f *framework.Framework, profile string) api.PodStatus {
 	pod := createPodWithAppArmor(f, profile)
-	framework.ExpectNoError(f.WaitForPodNoLongerRunning(pod.Name))
+	// The pod needs to start before it stops, so wait for the longer start timeout.
```

Comment:
Just for my own education, all apparmor tests expect the pod to fail in certain way?